### PR TITLE
HBCA-5 Integrate Autocomplete With Layout

### PR DESCRIPTION
**Pull request by heathharrelson**
Integrate the autocomplete input with Liz's layout from `mockups/search.html`.

### CODE REVIEW

File: test/unit/specs/components/SearchInput.spec.js [hunk 1]
```diff
@@ -1,35 +1,13 @@
 import { shallow } from '@vue/test-utils';
-import SearchForm from '@/components/SearchForm';
+import SearchInput from '@/components/SearchInput';
+
+import exampleResponses from '../../example-responses';
 
 // Stub out debounce to work around Jest timer issue:
 // https://github.com/facebook/jest/issues/3465
 jest.mock('lodash/debounce', () => jest.fn(fn => fn));
```
Comment (heathharrelson) [inline test/unit/specs/components/SearchInput.spec.js]: Extracted for use in different tests.

File: src/App.vue [hunk 2]
```diff
@@ -1,7 +1,22 @@
 <template>
-  <div id="app">
-    <div class="grid-container">
-      <router-view/>
```
Comment (heathharrelson) [inline src/App.vue]: I assume the header will get extracted into a component at some point.

File: src/App.vue [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
+  <div id="app" class="wrapper">
+    <div>
+      <div>
+        <div class="top-bar header">
+         </div>
+        <nav class="second-bar nav-bar-wrapper">
```
Comment (wsams) [inline src/App.vue]: Looks like some tabbing got offset here if you want to update that.

File: src/components/QuestionsPage.vue [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+      </div>
+
+      <div class="saved-terms">
+        <span v-for="(selection, index) in selections"
+              :key="selection.id"
+              class="label symptom-tag">{{ selection.exact_synonym[0] }} <strong @click="removeSelection(index)">X</strong></span>
+      </div>
+
+      <div class="button-container ">
```
Comment (wsams) [inline src/components/QuestionsPage.vue]: Extra space after container

File: src/components/SearchInput.vue [hunk 5]
```diff
@@ -56,7 +57,7 @@ export default {
      * @param {Number} index - the index in the `suggestions` array of the item selected.
      */
     selectItem(index) {
-      this.selections.push(this.suggestions[index]);
+      this.$emit('itemSelected', this.suggestions[index]);
```
Comment (wsams) [inline src/components/SearchInput.vue]: Had to look this up. So this pushes up to `QuestionsPage`?

Comment (heathharrelson) [reply src/components/SearchInput.vue]: Exactly. This sends a custom event that `QuestionsPage` listens for and uses to capture the selected term. It's basically equivalent to an Ember action.

**Pull request merged**